app/javascript/.storybook/main.js

```diff
@@ -10,6 +10,16 @@ module.exports = {
     '@storybook/addon-links',
     '@storybook/addon-a11y',
     '@storybook/addon-notes/register-panel',
+    {
+      name: '@storybook/addon-storysource',
+      loaderOptions: {
+        prettierConfig: {
```

Comment:
As a follow up PR, we could pull out the prettier config from package.json and make it it's own file which could be imported here so that we do not need to maintain two prettier configurations.

Reply:
Awesome. I'll create a pr for that.